man/Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM    golang:1.6.3-alpine
+FROM    alpine:3.4
 
-RUN     apk add -U git bash curl gcc musl-dev
+RUN     apk add -U git go bash curl gcc musl-dev
 
-RUN     export GLIDE=0.10.2; \
-        export SRC=https://github.com/Masterminds/glide/releases/download/; \
-        curl -sL ${SRC}/${GLIDE}/glide-${GLIDE}-linux-amd64.tar.gz | \
-        tar -xz linux-amd64/glide && \
-        mv linux-amd64/glide /usr/bin/glide && \
+RUN     mkdir -p /go/src /go/bin /go/pkg
+ENV     GOPATH=/go
+RUN     export GLIDE=v0.11.1; \
+        export SRC=https://dl.bintray.com/dnephin/glide/; \
+        curl -sL -o /usr/bin/glide ${SRC}/glide-${GLIDE}-linux-amd64 && \
```

Comment:
Ok, but we are talking about getting an upstream project to generate builds for eventually ppc64le, s390x, arm64, arm, and more... in order to build absolutely identical man pages. It really makes vendoring seem like a better option.